packages/api/src/external/aws/address.ts

```diff
@@ -11,29 +11,44 @@ const indexName = Config.getPlaceIndexName();
 const placeIndexRegion = Config.getPlaceIndexRegion();
 const client = makeLocationClient(placeIndexRegion);
 
+export function buildAddressText(address: Address): string {
+  return `${address.addressLine1}, ${address.city}, ${address.state} ${address.zip}`;
+}
+
 /**
- * Geocodes a list of addresses using Amazon Location Services.
- * @param addresses
- * @returns
+ * Adds coordinates to addresses that don't already have them
+ *
+ * @param addresses a list of Address objects
+ * @returns a boolean indicating whether any addresses were updated
  */
-export async function geocodeAddresses(addresses: Address[]): Promise<Coordinates[]> {
-  const resultPromises = await Promise.allSettled(
+export async function addGeographicCoordinates(addresses: Address[]): Promise<boolean> {
```

Comment:
I think this function needs a slight rework, where it will return back the addresses, so it's more clear what's going on in the places where it's called. 

I'm thinking to change it so it returns { address: Address[], updates: boolean }, so it also returns a boolean indicating whether any addresses got updated or not. This can be used upstream to decide whether the patient data needs to be updated.

Thoughts?